### PR TITLE
update the pages 'available questionnaires for a group' and 'impersonate'

### PR DESCRIPTION
**Pull request by lcarcone**
I have added the Material icon 'Add'

Thanks

### CODE REVIEW

File: css/site-wbs-style.css [hunk 1]
```diff
@@ -456,15 +475,15 @@ span.qaire {
 }
 
 span.qaire.votable {
- margin-left:-1em;
+ // margin-left:-1em;
```
Comment (tripu) [inline css/site-wbs-style.css]: Is `//` valid for CSS comments? I thought only `/* */` was legal&hellip;

File: css/site-wbs-style.css [hunk 2]
```diff
@@ -456,15 +475,15 @@ span.qaire {
...
 }
 
 span.qaireTools {
   display:block;
 }
 
 .votable:before {
-  content:"\0bb";
+ // content:"\0bb";
```
Comment (tripu) [inline css/site-wbs-style.css]: Is `//` valid for CSS comments? I thought only `/* */` was legal&hellip;

File: html/qtypes-site-current.html [hunk 3]
```diff
@@ -86,14 +86,25 @@ <h1 class="title">open questionnaires for the Test Group</h1>
...
-</ul><ul>
-<li><a href='factory'><img src='/2002/09/wbs/icons/create' alt='new' />Add a new questionnaire</a></li><li>List <a href='all'>all</a> / <a href='closed'>closed</a> questionnaires for this group</li></ul>
+<li><span class='qaire votable'><a href='test-call/'>Call for Review:
+  HTML tidy service is W3C Proposed Recommendation</a>: open until 29
+  July 2016</span><span class='qaireTools'>&nbsp; <a class='icon'
+  href='test-call/handler' title='Edit &quot;Call for Review: HTML
+  tidy service is W3C Proposed Recommendation&quot;'><img
+  src='../icons/stock_edit.png' alt='Edit' /></a> &nbsp; <a
+  class='icon' href='test-call/results' title='Results of &quot;Call for Review: HTML tidy service is W3C Proposed Recommendation&quot;'><img src='../icons/list.png' alt='List' /></a> </span><br /></li>
```
Comment (tripu) [inline html/qtypes-site-current.html]: I like our new icons, but sometimes they're not intuitive enough.

`icons/stock_edit.png` clearly means &ldquo;edit&rdquo;. But `icons/list.png` doesn't scream &ldquo;results&rdquo;.

I don't think we'd find an icon for &ldquo;results&rdquo; that is clear for all users. So, maybe add a label next to it&hellip;?

**Review by tripu — CHANGES_REQUESTED** [review]
LGTM if we address these comments.

File: html/qtypes-site-current.html (hunk 3 above)
Comment (lcarcone) [reply html/qtypes-site-current.html]: @tripu I fixed the css and added a title for icons Edit/Results.
Is it ok for you ?

**Review by tripu — APPROVED** [review]

**Comment by tripu** [comment]
Thank you, @lcarcone!

**Pull request merged**